cmd/proxy/main.go

```diff
@@ -302,6 +301,7 @@ func (p *Proxy) runUDSMasterServer(ctx context.Context, o *ProxyRunOptions, serv
 		}
 		go grpcServer.Serve(lis)
 		stop = grpcServer.GracefulStop
+		klog.Info("runUDSMasterServer in GRPC mode")
```

Comment:
I like adding this log message. We should add equivalent log messages for the other configurations.
I dislike negative logic and do not want to rely on if it doesn't say GRPC mode then ...